docs/modules/ROOT/pages/component-loki.adoc

```diff
@@ -0,0 +1,29 @@
+= Loki
+
+[abstract]
+Loki can be used to process and store Couchbase logs.
+
+== Overview
+
+https://grafana.com/oss/Loki/[Loki^] is the https://prometheus.io/[Prometheus^]-inspired open source, multi-tenant log aggregation system.
+Loki is used to process and store collected Couchbase logs.
+
+ifdef::env-github[]
+:imagesdir: https://github.com/couchbaselabs/observability/raw/main/docs/modules/ROOT/assets/images
+endif::[]
+image:component-loki.png[]
+
+Couchbase cluster generates logs of various operations which is important for monitoring and investigation purposes.
+Loki provides the option to parse the logs enabling search and aggregation based on keys.
+Something needs to forward the logs, we recommend Fluent Bit and provide a method to deploy it.
```

Comment:
Something sounds a bit weird, I was just using it as an example. Maybe reword "Logs are pushed to Loki by various tools, we recommend...."